server/src/logic.js

```diff
@@ -225,21 +225,27 @@ export const groupHandler = {
   createGroup(_, args, ctx) {
     const { name } = args.group;
     return getAuthenticatedUser(ctx).then(user =>
-      Creators.group({ name, user }),
+      Creators.group({ name, user, organisation: 1 }), //TODO: dont be static
     );
   },
   addUserToGroup(_, args, ctx) {
-    return getAuthenticatedUser(ctx).then(() =>
-      Group.findById(args.groupUpdate.groupId).then((group) => {
+    const { userId,groupId } = args.groupUpdate;
```

Comment:
all of this stuff needs to be in a seperate pull request and come with tests imo